### PR TITLE
adpupa prepbufr yaml dimension fix

### PR DESCRIPTION
**Pull request by PraveenKumar-NOAA**
## Description

This PR updates dimension of variables into the adpupa prepbufr yaml and removes extra copies of the yaml/nc files for this data type. 

The following two files were modified for the dimension updates: 
- testinput/bufr_ncep_prepbufr_adpupa.yaml
- testoutput/bufr_ncep_prepbufr_adpupa.nc

The following two files were removed: 
- testinput/adpupa_prepbufr.yaml
- testoutput/adpupa_prepbufr.nc

The following file is also modified to remove the unit test for the adpupa_prepbufr.yaml: 
- CMakeLists.txt

## Dependencies

None

## Impact

None

### CODE REVIEW

File: test/testoutput/bufr_ncep_prepbufr_adpupa.nc
Comment (PatNichols) [inline test/testoutput/bufr_ncep_prepbufr_adpupa.nc:1]: @PraveenKumar-NOAA Two comments. 1) The file is fairly large, >1MB. Can the size be reduced. The second is that the name is a little redundant. Could it be ncep_prepbur_adupa.nc or something similar? The last is a minor point.

Comment (PraveenKumar-NOAA) [reply test/testoutput/bufr_ncep_prepbufr_adpupa.nc:1]: @PatNichols why all the three checks are failing, any idea?

**Comment by PatNichols** [comment]
@PraveenKumar-NOAA I am not sure you can see the output from your ctests since cdash is failing at the moment:
The following tests FAILED:
--
1215 | 115 - test_iodaconv_bufr_ncep_prepbufr_adpupa (Failed)
1216 | 116 - test_iodaconv_bufr_ncep_prepbufr_adpupa_api (Failed)

**Comment by PraveenKumar-NOAA** [comment]
> ## @PraveenKumar-NOAA I am not sure you can see the output from your ctests since cdash is failing at the moment:
> The following tests FAILED:
> 1215 | 115 - test_iodaconv_bufr_ncep_prepbufr_adpupa (Failed) 1216 | 116 - test_iodaconv_bufr_ncep_prepbufr_adpupa_api (Failed)

@PatNichols thanks, I will check once HERA is up for work.

**Comment by PraveenKumar-NOAA** [comment]
> > ## @PraveenKumar-NOAA I am not sure you can see the output from your ctests since cdash is failing at the moment:
> > The following tests FAILED:
> > 1215 | 115 - test_iodaconv_bufr_ncep_prepbufr_adpupa (Failed) 1216 | 116 - test_iodaconv_bufr_ncep_prepbufr_adpupa_api (Failed)
> 
> @PatNichols thanks, I will check once HERA is up for work.

@PatNichols this problem is arising after I trimmed the input bufr file. Is it OK to proceed with the original ADPUPA.prepbufr file, which leads to the output NC file > 1 Mb?

**Comment by PraveenKumar-NOAA** [comment]
@emilyhcliu @rmclaren please review this PR.

**Comment by PatNichols** [comment]
@PraveenKumar-NOAA is it possible to reduce the test file size?

**Comment by PraveenKumar-NOAA** [comment]
> @PraveenKumar-NOAA is it possible to reduce the test file size?

@PatNichols No, I tried to reduce the size for different capacity, but it didn't work. We have to keep the same size.

**Review by rmclaren — APPROVED** [review]
Thanks for deleting the uneeded test and cleaning up this file.

**Review by emilyhcliu — APPROVED** [review]

**Comment by PatNichols** [comment]
> > @PraveenKumar-NOAA is it possible to reduce the test file size?
> 
> @PatNichols No, I tried to reduce the size for different capacity, but it didn't work. We have to keep the same size.

@PraveenKumar-NOAA Does it fail the test with a smaller file or does the reduction fail to produce a usable file?

**Comment by PraveenKumar-NOAA** [comment]
> > > @PraveenKumar-NOAA is it possible to reduce the test file size?
> > 
> > 
> > @PatNichols No, I tried to reduce the size for different capacity, but it didn't work. We have to keep the same size.
> 
> @PraveenKumar-NOAA Does it fail the test with a smaller file or does the reduction fail to produce a usable file?

@PatNichols test is failed with a smaller file.

**Comment by PatNichols** [comment]
> > > > @PraveenKumar-NOAA is it possible to reduce the test file size?
> > > 
> > > 
> > > @PatNichols No, I tried to reduce the size for different capacity, but it didn't work. We have to keep the same size.
> > 
> > 
> > @PraveenKumar-NOAA Does it fail the test with a smaller file or does the reduction fail to produce a usable file?
> 
> @PatNichols test is failed with a smaller file.

@PraveenKumar-NOAA Is it the case that reducing the netcdf file require the bufr file to be reduced and that is not easy?

**Comment by PraveenKumar-NOAA** [comment]
> > > > > @PraveenKumar-NOAA is it possible to reduce the test file size?
> > > > 
> > > > 
> > > > @PatNichols No, I tried to reduce the size for different capacity, but it didn't work. We have to keep the same size.
> > > 
> > > 
> > > @PraveenKumar-NOAA Does it fail the test with a smaller file or does the reduction fail to produce a usable file?
> > 
> > 
> > @PatNichols test is failed with a smaller file.
> 
> @PraveenKumar-NOAA Is it the case that reducing the netcdf file require the bufr file to be reduced and that is not easy?

@PatNichols yes, that is true.

**Comment by PraveenKumar-NOAA** [comment]
@PatNichols do you have any other question/ issue?

**Comment by PatNichols** [comment]
@PraveenKumar-NOAA We are just looking at the file sizes. I suppose it's not as big as an issue since you are removing on ~1.5 Mb and adding another. @fcvdb @BenjaminRuston What do you guys think?

**Pull request merged**